src/main/java/no/finn/unleash/repository/ToggleBootstrapHandler.java

```diff
@@ -43,7 +43,7 @@ public ToggleCollection read() {
         return new ToggleCollection(Collections.emptyList());
     }
 
-    private static class ToggleBootstrapRead implements UnleashEvent {
+    public static class ToggleBootstrapRead implements UnleashEvent {
```

Comment:
I had to make this public so the event could be introspected in the test (I can't imagine we want the event to not be visible to subscribers.)